lawnchair/res/values/strings.xml

```diff
@@ -60,6 +60,8 @@
     <string name="transparent_background_icons">Transparent Themed Icons</string>
     <string name="transparent_background_icons_description">Use transparent background on themed icons</string>
     <string name="smartspace_mode_selection">At a Glance Provider selection</string>
+    <string name="pref_separate_recents_activity_title">Separate Recents Activity</string>
```

Comment:
Switch preference options should ideally be in `<verb>` `<what to do>`. Also no one will need to know the internal code term, so I've clarified it as well by using "GestureNavContract":

```suggestion
    <string name="pref_separate_recents_activity_title">Enable GestureNavContract</string>
```